### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by CindyDorantes**
## Project requirements 🎯  🎯 
- [x]  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
    -  Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- [x] Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- [x] Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -4,6 +4,35 @@ def index
...
+        else
+          # error message
+          flash.now[:notice] = "Error: Couldn't create post"
+          # render new
+          render :new, locals: { post: }
+        end
+      end
+    end
+  end
+
```
Comment (1GORDON) [inline app/controllers/posts_controller.rb:35]: Great job @CindyDorantes 🤝

- [x] [Optional] You may also add a params method to the posts_controller. You can learn more about strong parameters by following this [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class CommentsController < ApplicationController
+  def new
+    comment = Comment.new
+    respond_to do |format|
+      format.html { render :new, locals: { comment: } }
+    end
+  end
+
```
Comment (1GORDON) [inline app/controllers/comments_controller.rb:8]: Great job @CindyDorantes 🤝

- [x] [Optional] You may also add a params method to the comments_controller. 
params-comments
You can learn more about strong parameters by following this [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @CindyDorantes ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (CindyDorantes) [reply app/controllers/posts_controller.rb:35]: sure! Thanks for the comment

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (CindyDorantes) [reply app/controllers/comments_controller.rb:8]: Of course! Thanks

**Pull request merged**